ddl/ddl_api.go

```diff
@@ -606,10 +613,17 @@ func checkGeneratedColumn(colDefs []*ast.ColumnDef) error {
 	return nil
 }
 
-func checkTooLongColumn(colDefs []*ast.ColumnDef) error {
-	for _, colDef := range colDefs {
-		if len(colDef.Name.Name.O) > mysql.MaxColumnNameLength {
-			return ErrTooLongIdent.GenWithStackByArgs(colDef.Name.Name)
+func checkTooLongColumn(cols []interface{}) error {
```

Comment:
How about create a new function `checkTooLongColumnName(cols []model.CIStr)` to do this？

Reply:
Same check logical with different input parameters, so I think rewrite existing `checkTooLongColumn` is better.